lib/assert.js

```diff
@@ -118,10 +119,12 @@ x.doesNotThrow = function (fn, msg) {
 	try {
 		assert.doesNotThrow(fn, msg);
 	} catch (err) {
-		test(false, create(err.actual, err.expected, err.operator, err.message, x.doesNotThrow));
+		test(false, create(err.actual, err.expected, err.operator, err.message, x.notThrows));
 	}
 };
 
+x.doesNotThrow = util.deprecate(x.notThrows, 't.doesNotThrow is renamed to t.notThrows. The old name still works, but will be removed in AVA 1.0.0. Update your references.');
```

Comment:
Can you alias it first, then deprecate `x.doesNotThrow` (not `x.notThrows) later?